Rakefile

```diff
@@ -1,7 +1,32 @@
 require 'rspec/core/rake_task'
 require 'foodcritic'
+require 'kitchen'
 
 FoodCritic::Rake::LintTask.new
 RSpec::Core::RakeTask.new(:rspec)
 
-task default: [:foodcritic, :rspec]
+desc "Run kitchen tests sequentially"
```

Comment:
OPT: factorize sequential and concurrent code using an concurrency enviroment variable.